go.mod

```diff
@@ -54,6 +54,10 @@ replace (
 	k8s.io/sample-controller => k8s.io/sample-controller v0.34.2
 )
 
+replace github.com/longhorn/go-common-libs => github.com/c3y1huang/go-common-libs v0.0.0-20251121102418-2934b0a57cd0
+
+replace github.com/henrygd/beszel => github.com/c3y1huang/beszel v0.0.0-20251119090922-8c70e799d095
```

Comment:
@derekbit , we need to fork https://github.com/henrygd/beszel to remove the embedded Windows binaries so Longhorn can resolve the module during vendoring.
- https://github.com/c3y1huang/beszel/commit/e2173159cb6d48a0b976bb77ad4414a497ad2421
- https://github.com/c3y1huang/beszel/commit/8c70e799d095e8675d2064c5c3dccfb34ce5a956

Reply:
Forked to https://github.com/longhorn/beszel.